.github/workflows/emscripten.yml

```diff
@@ -1,5 +1,8 @@
 name: Emscripten
 
+env:
+  EMSCRIPTEN_SDK_VERSION: 2.0.30
```

Comment:
I would suggest importing it from a separate file that can be shared with other submodules.

Reply:
Yea. Perhaps it should be done by analogy with the file `configure_cmake.sh`